frontend/src/components/MeetingRoom/ToolbarOverflowButton/ToolbarOverflowButton.tsx

```diff
@@ -3,11 +3,13 @@ import { Tooltip } from '@mui/material';
 import MoreVertIcon from '@mui/icons-material/MoreVert';
 import ToolbarButton from '../ToolbarButton';
 import ToolbarOverflowMenu from '../ToolbarOverflowMenu';
+import UnreadMessagesBadge from '../UnreadMessagesBadge';
 
 /**
  * ToolbarOverflowButton Component
  *
- * Displays a clickable button that opens a grid of hidden toolbar buttons for smaller viewport devices.
+ * Displays a clickable button that opens a grid of hidden toolbar buttons for smaller viewport devices. There
+ * is an unread chat messages indicator that is shown when there are messages to be viewed.
```

Comment:
Nit: "There is" -> "There is also"?
Another nit: "to be viewed" -> "to be read"?